### DISCUSSION TITLE
Client 3.416 Server 3.451

### DISCUSSION THREAD

**Discussion by Onair-santa** [vpnhood/VpnHood#447] [General]
Will client 3.416 work with the new server 3.451? Or do I need to disable server updates? Thnx

**Comment by trudyhood** [comment]
Hi, 416 is pretty old. There are pretty good and essential updates. I think you should test it in a different environment. 
However, if you stick with a specific client version (perhaps your own), you should disable auto updates. We need to upgrade both clients and servers. If one of them becomes too old, it may be out of sync and unable to be upgraded.

We are committed to removing old code to maintain a clean and organized codebase. We regret to inform you that we cannot provide long-term support for an old code base.
Try to keep your apps up to date.

**Comment by trudyhood** [comment]
Please don't rely on our pre-release. It appears to have issues with older versions. Also, I'm not sure what you mean by hiding process. There is nothing special in v416.
